python/src/tasks/embedding.py

```diff
@@ -27,10 +26,6 @@ def compute(self):
             data=json.dumps(request),
         )
 
+        # The index order relies on cells_id in an ascending form. The order is made in the R part. 
         result = r.json()
-
-        # order cells by cell ids first to guarantee order
-        sorted_indices = self.adata.obs.sort_values(by=["cell_ids"]).index
-        self.adata = self.adata[sorted_indices, :]
```

Comment:
I had left this as a reminder of the correct order we are expecting, it's good that you caught this one.